packages/node/src/index.ts

```diff
@@ -17,7 +17,22 @@ export default class LogtoClient extends BaseClient {
   constructor(config: LogtoConfig, adapter: Pick<ClientAdapter, 'navigate' | 'storage'>) {
     super(config, {
       ...adapter,
-      requester: createRequester(fetch),
+      requester: createRequester(
+        config.appSecret
+          ? async (...args: Parameters<typeof fetch>) => {
+              return fetch(args[0], {
+                ...args[1],
```

Comment:
What are `args[0]` and `args[1]` specifically? `url` and `configs`?
Would it be better to de-construct them first?
```ts
const [url, configs] = args;
```

Reply:
updated

<img width="645" alt="image" src="https://user-images.githubusercontent.com/5717882/182327953-99b39d76-92af-4fb4-9d10-ed4e5c903355.png">